petastorm/reader.py

```diff
@@ -550,6 +556,13 @@ def _partition_row_groups(self, dataset, row_groups, shard_count, cur_shard,
 
         # We hash on the relative path of each parquet file to guarantee consistency between different reader
         # constructions even after moving the dataset
+        if shard_seed:
+            import random
```

Comment:
Do you know why these lines appear as not covered? Is anything wrong with either the test or the code?

Reply:
hmmm, I modified one unit test to cover `shard_seed` != None case. Let me double check.

Reply:
Should be fixed. I used `0` for shard_seed, which make `if shard_seed` false.